Projects/Contracts/Sources/Reusables/ContractRow.swift

```diff
@@ -11,26 +11,10 @@ struct ContractRow: Hashable {
     static func == (lhs: ContractRow, rhs: ContractRow) -> Bool { lhs.hashValue == rhs.hashValue }
 
     func hash(into hasher: inout Hasher) {
-        hasher.combine(displayName)
-        hasher.combine(statusPills)
-        hasher.combine(isContractActivated)
-        hasher.combine(detailPills)
+        hasher.combine(contract.id)
```

Comment:
I think only using the ID here would make this view not `rerender` correctly right? Maybe make the model washable and then you can just `hasher.combine(contract)`.

Reply:
Do I need detergent to make it washable?

Reply:
Also not sure why it wouldn't render properly?